README.md

```diff
@@ -1044,6 +1045,43 @@ Below is the complete list of available options that can be used to customize yo
 | `RACK_ATTACK_BANTIME` | Number of seconds an IP should be banned after too many auth attempts. Defaults to `3600`. |
 | `GITLAB_WORKHORSE_TIMEOUT` | Timeout for gitlab workhorse http proxy. Defaults to `5m0s`. |
 
+### Docker secrets and configs
+
+All the above environment variables can be put into a [secrets](https://docs.docker.com/compose/compose-file/#secrets) or [config](https://docs.docker.com/compose/compose-file/#configs) file
+and then both docker-compose and Docker Swarm can import them into your gitlab container.
+
+On startup, the gitlab container will source env vars from a config file named `gitlab-config`, and then a secrets file named `gitlab-secrets` (both mounted in the default locations).
+
+To modify your existing `docker-compose.yml` file, add the following to your gitlab service:
+
+> Note: you also need to change the docker-compose file version `version: "3.4"`.
+
+```
+services:
+  gitlab:
+
+...
+
+    configs:
+      - gitlab-configs
+    secrets:
+      - gitlab-secrets
+```
+
+And then define those secrets and configs:
+
+```
+configs:
+  gitlab-configs:
+    file: ./gitlab.configs
+
+secrets:
+  gitlab-secrets:
+    file: ./gitlab.secrets
+```
+
+If you're only using one of these files, don't include its entries in the docker-compose file.
```

Comment:
Does this mean *"If you're not using one of these files, then don't include its entry in the docker-compose file"*? It's a little confusing as currently worded.